### PR TITLE
DEPRECATED: Fix `warning: "L" redefined`

### PR DESCRIPTION
**Pull request by Blackhex**
This PR fixes:

```
../../../gcc/libgcc/config/aarch64/lse.S:127:11: warning: "L" redefined
  127 | #  define L    l
      |           ^
In file included from ../../../gcc/libgcc/config/aarch64/lse.S:54:
../../../gcc/libgcc/config/aarch64/aarch64-asm.h:27:9: note: this is the location of the previous definition
   27 | #define L(label) .L ## label
      |         ^
```
when bulding LSE intrinsics objects.

### CODE REVIEW

**Review by vejbomar — APPROVED** [review]

File: libgcc/config/aarch64/lse.S [hunk 1]
```diff
@@ -52,6 +52,8 @@ see the files COPYING3 and COPYING.RUNTIME respectively.  If not, see
  */
 
 #include "aarch64-asm.h"
+#undef L
```
Comment (eukarpov) [inline libgcc/config/aarch64/lse.S:55]: It has strange usage of L in this file. I would remove all "L" definitions from this file.

Comment (Blackhex) [reply libgcc/config/aarch64/lse.S:55]: This was addressed upstream.

Comment (eukarpov) [reply libgcc/config/aarch64/lse.S:55]: Could you please provide a link?

Comment (Blackhex) [reply libgcc/config/aarch64/lse.S:55]: [aarch64: libgcc: Cleanup warnings in lse.S ·](https://github.com/Windows-on-ARM-Experiments/gcc-woarm64/commit/e4d417417f8a2f1cf615b6ff81687f4861787de0) e4d4174

Comment (eukarpov) [reply libgcc/config/aarch64/lse.S:55]: Thanks. Here is a patch series.
https://gcc.gnu.org/pipermail/gcc-patches/2024-October/664433.html

Why this PR is needed? Should not it be fixed by rebasing?

Comment (Blackhex) [reply libgcc/config/aarch64/lse.S:55]: It was needed before this change has been made upstream. That's why I commented that the change has been done upstream. I will close this issue once the mentioned change will be in `woarm64` branch.

**Comment by Blackhex** [comment]
Closing as it was fixed upstream.

**Pull request closed without merging**